src/components/molecules/DropDown/index.tsx

```diff
@@ -1,7 +1,164 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import Styled from 'styled-components';
+import arrowDown from 'assets/images/arrowDown.svg';
+import arrowUp from 'assets/images/arrowUp.svg';
 
-function DropDown(): React.ReactElement {
-  return <div>DropDown</div>;
+export interface IProps {
+  className?: string;
+  setState: (value: string) => void;
+  state: string;
+  defaultMsg: string;
+  itemList: string[];
+  page: string;
 }
 
+function DropDown({ setState, state, defaultMsg, itemList, page }: IProps): React.ReactElement {
+  const [isOpen, setIsOpen] = useState<boolean>(false); //드롭다운이 열렸는지
+  const [isChecked, setIsChecked] = useState<boolean>(false); //값이 선택이 되었는지
+  const radioClickListener = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setState(e.target.value);
+    setIsOpen(false);
+  };
+  const openClickListener = (): void => {
+    setIsOpen(!isOpen);
+  };
+
+  useEffect(() => {
+    if (state !== defaultMsg) {
+      setIsChecked(true);
+    } else {
+      setIsChecked(false);
+    }
```

Comment:
이부분 이렇게 정리하면 조금 더 깔끔할 것 같네요!
```suggestion
      setIsChecked(!(state !== defaultMsg));
```